single-node-refactor/src/common/io_utils.h

```diff
@@ -97,7 +158,6 @@ class MeshReader
     ///
     /////////////////////////////////////////////////////////////////////////////
     void read_mesh(mesh_t& mesh, 
-                   MaterialPoint_t& MaterialPoints, 
```

Comment:
What drove the removal of material points here?

Reply:
the mesh doesn't have material info